server/types.go

```diff
@@ -0,0 +1,6 @@
+package server
```

Comment:
how about a new package called `networkModel`? -- that would be any structs that only the network needs that don't exist in model

Reply:
👍